src/client/ClientDataResolver.js

```diff
@@ -360,6 +361,7 @@ class ClientDataResolver {
    */
   static resolveColor(color) {
     if (typeof color === 'string') {
+      if (color === 'RANDOM') return Math.floor(Math.random() * 16777214) + 1;
```

Comment:
i had a reason that i made it 1, but i don't remember what it was. if i can't remember it in four hours from this posting i will make it 0